packages/google_maps_flutter/example/lib/map_data_ui.dart

```diff
@@ -0,0 +1,98 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:google_maps_flutter/google_maps_flutter.dart';
+import 'package:google_maps_flutter_example/page.dart';
```

Comment:
you want to import this as `import 'page.dart';`

Reply:
done